Feature/ui

tjbenton opened the pull request.

alex-andreescu commented on scripts/index-files.js:
```diff
@@ -2,6 +2,7 @@
 
 const { Files } = require('../utils/files')
 
+
```
Why the extra empty line here?

tjbenton replied:
there's supposed to be 2 blank lines after the import/require

tjbenton committed:
New: UI config

tjbenton committed:
Publish

 - @ma-shop/locale@0.0.18
 - @ma-shop/react-native@0.0.23
 - @ma-shop/utils@0.0.21
